src/System.Data.SqlClient/src/Interop/ILocaleApiHelper.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Data
+{
+    internal interface ILocaleApiHelper
```

Comment:
Can we not use the interface here and go with an abstract class? Generally following the overall philosophy.